sobit_light_pose_list:
  ros__parameters:
    initial_pose:
      arm_shoulder_roll  : 0.0
      arm_shoulder_pitch : 1.5708
      arm_elbow_pitch    : 0.0
      arm_forearm_roll   : 0.0
      arm_wrist_pitch    : 0.0
      arm_wrist_roll     : 0.0
      hand               : 0.0
      head_yaw           : 0.0
      head_pitch         : 0.0

    detecting_pose:
      arm_shoulder_roll  : 0.0
      arm_shoulder_pitch : 1.5708
      arm_elbow_pitch    : 0.0
      arm_forearm_roll   : 0.0
      arm_wrist_pitch    : 0.0
      arm_wrist_roll     : 0.0
      hand               : 0.0
      head_yaw           : 0.0
      head_pitch         : 0.0

    following_pose:
      arm_shoulder_roll  : 0.0
      arm_shoulder_pitch : 1.5708
      arm_elbow_pitch    : 0.0
      arm_forearm_roll   : 0.0
      arm_wrist_pitch    : 0.0
      arm_wrist_roll     : 0.0
      hand               : 0.0
      head_yaw           : 0.0
      head_pitch         : 0.0
